### PR TITLE
IR-4688 Merge chargebee setting to engine settings

### PR DESCRIPTION
**Pull request by mshafiqmk**
## Summary

Description: This PR migrates the Chargebee settings from the **chargebee-setting** table to the **engine-setting** table. The following changes have been made:

### Migration Script:

Created a new migration script to move existing data from **chargebee-setting** to **engine-setting**.
The migration script also drops the chargebee-setting table after the data has been moved.

### Code Update:

Updated **updateAppConfig.ts** to read and write Chargebee settings from the engine-settings table instead of the chargebee-setting table.

### Seed File Update:
Updated the **engine-setting** seed file to insert Chargebee settings into the engine-settings table.

## References
closes #_insert number here_

### CODE REVIEW

**Comment by mshafiqmk** [comment]
> **Thank you for your pull request and welcome to the iR Engine developer community!**
> 
> We require contributors to sign our Copyright Assignment Agreement, and we don't seem to have you on file.
> 
> In order for us to review and merge your code, please sign the agreement at https://forms.gle/stb3bi9X2K7RvqT66
> 
> The agreement has not been signed by users: @mshafiqmk.
> 
> After signing the agreement, you can ask me to recheck this PR by posting `@cla-bot check` as a comment to the PR :)

@cla-bot can you please check my access.

**Review by barankyle — APPROVED** [review]

File: packages/server-core/src/updateAppConfig.ts [hunk 1]
```diff
@@ -141,13 +139,21 @@ export const updateAppConfig = async (): Promise<void> => {
...
+        },
+        {} as Record<string, string>
+      )
+
+      appConfig.chargebee = {
+        ...appConfig.chargebee,
+        url: chargebeeSettings[EngineSettings.Chargebee.Url],
+        apiKey: chargebeeSettings[EngineSettings.Chargebee.ApiKey]
       }
     })
```
Comment (hanzlamateen) [inline packages/server-core/src/updateAppConfig.ts]: This needs to be done in https://github.com/ir-engine/ir-engine/blob/5213e54f39c914040a3f739c49d2883531f96dc5/packages/server-core/src/setting/engine-setting/engine-setting.ts#L62

**Review by hanzlamateen — CHANGES_REQUESTED** [review]

File: packages/server-core/src/updateAppConfig.ts (hunk 1 above)
Comment (mshafiqmk) [reply packages/server-core/src/updateAppConfig.ts]: @hanzlamateen done.

Comment (hanzlamateen) [reply packages/server-core/src/updateAppConfig.ts]: `chargebeeSettingPromise` needs to be remove from updateAppConfig.ts

**Review by hanzlamateen — CHANGES_REQUESTED** [review]

**Review by hanzlamateen — APPROVED** [review]

**Pull request merged**